blueocean-dashboard/src/main/js/util/UrlUtils.js

```diff
@@ -46,9 +46,17 @@ export const buildRunDetailsUrl = (organization, pipeline, branch, runId, tabNam
 /*
  * helper to clean the path replace(/%2F/g, '%252F')
  * @param input
+ *
+ * XXX: This was used where encodeURIComponent() should have been used. If it has not other uses,
+ *      it should perhaps be removed.
  */
 export const uriString = (input) => encodeURIComponent(input).replace(/%2F/g, '%252F');
 
+/**
+ * Double encode name, feature/test#1 is encoded as feature%252Ftest%25231
+ */
+export const doubleUriEncode = (input) => escape(encodeURIComponent(input));
```

Comment:
`escape` is deprecated. This should use `encodeURIComponent` twice.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape